src/store/modules/permissions.js

```diff
@@ -41,8 +41,8 @@ export default {
   namespaced: true,
 
   state: {
-    address: true,
-    messageSign: true,
+    address: false,
+    messageSign: false,
```

Comment:
That would mean turned off by default, right?

Reply:
yes, as they were before

Reply:
Shouldn't the permission checking logic be inverted as well? Or it wasn't working properly before? cc: @etharner

Reply:
@davidyuk Yes, it's inverted. I've asked for changes. I also propose to uninvert it in aepps permissions (this way false in store will mean "ask = _don't have_ permission) because it's messy to work with it